app/src/main/java/com/readme/android/adapter/MainViewPagerAdapter.kt

```diff
@@ -0,0 +1,11 @@
+package com.readme.android.adapter
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentActivity
+import androidx.viewpager2.adapter.FragmentStateAdapter
+
+class MainViewPagerAdapter(fa: FragmentActivity) : FragmentStateAdapter(fa) {
+    val fragments = mutableListOf<Fragment>()
```

Comment:
굳이굳이 캡슐화 원칙 지켜서 코드 짠다면.. 이렇게 짜도 괜춘할듯합니당..!
```suggestion
        private val _fragmentList = mutableListOf<Fragment>()
    var fragmentList: List<Fragment> = _fragmentList
        set(value) {
            _fragmentList.addAll(value)
        }
```

Reply:
이것두 반영완!~!